packages/xod-client/src/messages/actions.js

```diff
@@ -1,21 +1,32 @@
+import * as R from 'ramda';
 import * as ActionType from './actionTypes';
 import { MESSAGE_TYPE } from './constants';
 import { STATUS } from '../utils/constants';
 
 const getTimestamp = () => new Date().getTime();
 
-export const addMessage = (
-  type,
-  messageData,
-  persistent = false,
-  id = null
-) => ({
+const getDefaultPersistencyByType = type => {
```

Comment:
why not just `const getDefaultPersistencyByType = R.equals(MESSAGE_TYPE.ERROR)`?